build-system/common/utils.js

```diff
@@ -26,6 +26,15 @@ const {isTravisBuild} = require('./travis');
 
 const ROOT_DIR = path.resolve(__dirname, '../../');
 
+/**
+ * Cleans and builds binaries with --fortesting flag and
+ * overriden config.
+ */
+function buildRuntime() {
```

Comment:
Since this could be confused for `gulp build`, can you rename it to `buildMinifiedRuntime`?

Reply:
SG